docker-compose.yml

```diff
@@ -0,0 +1,39 @@
+# Copyright 2018-2020 ABSA Group Limited
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This docker-compose script is only for development purposes! Passwords should never be stored in plaintext!
+version: "3"
+
+services:
```

Comment:
Just a quick one. Don't you have to create a network to join services likes this? 
What I mean at docker [docs](https://docs.docker.com/compose/compose-file/#networks)

Reply:
Yeah no. A default network is created by docker-compose. https://docs.docker.com/compose/networking/